selfdrive/manager/manager.py

```diff
@@ -136,6 +134,10 @@ def manager_thread():
     pm_apply_packages('enable')
     start_offroad()
 
+  ensure_running(managed_processes.values(), False)
```

Comment:
Pass in the ignore list, otherwise it might start processes here that aren't supposed to run.